SummerSchool/app/src/main/java/nl/rug/www/rugsummerschools/controller/ThumbnailDownloader.java

```diff
@@ -0,0 +1,96 @@
+package nl.rug.www.rugsummerschools.controller;
+
+import android.os.Handler;
+import android.os.HandlerThread;
+import android.os.Message;
+import android.util.Log;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import nl.rug.www.rugsummerschools.networking.NetworkingService;
+
+/**
+ * Created by jk on 17. 12. 6.
+ */
+
+@Deprecated
+public class ThumbnailDownloader<T> extends HandlerThread {
+    private static final String TAG = "ThumbnailDownloader";
+    private static final int MESSAGE_DOWNLOAD = 0;
+
+    private boolean mHasQuit = false;
+    private Handler mRequestHandler;
+    private ConcurrentMap<T, String> mRequestMap = new ConcurrentHashMap<>();
+    private Handler mResponseHandler;
+    private ThumbnailDownloadListener<T> mThumbnailDownloadListener;
+
+    public interface ThumbnailDownloadListener<T> {
+        //TODO : need to change thumbnail
```

Comment:
add issue instead of leaving this as a code comment

Reply:
The class is for the future usage.
Also, it is reminder for me to implement what to do while working in general.
Specifically, this comment is not my priority to do and not an issue of the app yet.

Reply:
oki!